ctest-next/Cargo.lock

```diff
@@ -0,0 +1,485 @@
+# This file is automatically @generated by Cargo.
```

Comment:
This file shouldn't be needed since `ctest-next` is in the root workspace (and thus the root `Cargo.lock`)